### PR TITLE
Added initial package.json and bower.json files

### PR DESCRIPTION
**Pull request by stevenshuhyow**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Vacant Aardvark",
```
Comment (kevhuang) [inline bower.json]: should be Joseki

File: package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/VacantAardvark/joseki/issues"
+  },
+  "homepage": "https://github.com/VacantAardvark/joseki",
+  "dependencies": {
+    "body-parser": "1.12.3",
+    "bower": "1.4.1",
+    "express": "4.12.3",
+    "mysql": "5.6.24",
```
Comment (kevhuang) [inline package.json]: This doesn't install the actual MySQL database. We can exclude this from the package.json file. @dmsakamoto said we could use a MySQL add-on in Heroku.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "react": "^0.13.3",
```
Comment (kevhuang) [inline package.json]: I would take out the ^ so that we don't run into any deployment issues in case there's a new version

File: package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "sequelize": "2.1.0"
+  },
+  "devDependencies": {
```
Comment (kevhuang) [inline package.json:35]: Let's take out the tildes too

File: package.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Joseki",
+  "version": "1.0.0",
+  "description": "Easy way to organize events",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha test/server.spec.js",
+    "start": "gulp watch & nodemon server/server.js",
```
Comment (kevhuang) [inline package.json:8]: This is different than the "main" (index.js).

**Comment by dmsakamoto** [comment]
:+1:

**Comment by kevhuang** [comment]
looks good

**Comment by dmsakamoto** [comment]
:+1:

**Pull request merged**